bootstrap/opnsense-bootstrap.sh

```diff
@@ -108,9 +108,9 @@ echo "will automatically reboot."
 
 if [ -z "${DO_YES}" ]; then
 	echo
-	echo -n "Proceed with this action? [y/N]: "
-
-	read YN
+	echo "Proceed with this action? [y/N]: "
```

Comment:
these definitely look wrong as it moves the prompt to the next line

Reply:
Hey @fichtner, thanks for the quick response! Yeah, I agree that looks off. I think space actually was there last time, or at least that's what came up for me.

If you want to see what I was seeing with the linting, I used the shellcheck plugin in VS Code.

Reply:
No, I meant the dropping of "-n"

Reply:
I think the script told me that -n  that wasn't "POSIX" but worked in normal bash.

Reply:
True https://pubs.opengroup.org/onlinepubs/9699919799/utilities/echo.html but in this case we need the newline, which is recommended to be done by printf which doesn't make this any easier to read :D